stdlib/public/core/KeyPath.swift

```diff
@@ -26,11 +26,11 @@ internal func _abstract(
 
 // MARK: Type-erased abstract base classes
 
-/// A type-erased key path, from any root type to any resulting value
-/// type. NOTE: older runtimes had Swift.AnyKeyPath as the ObjC name.
```

Comment:
This first part *is* documentation.

Reply:
Argh. I guess if I must have a terminal case of the Dumb today, this is a decent place to have it. I'll fix that.